indico/modules/events/contributions/forms.py

```diff
@@ -32,3 +32,13 @@ class ContributionForm(IndicoForm):
     duration = TimeDeltaField(_("Duration"), [DataRequired()], default=timedelta(minutes=20),
                               units=('minutes', 'hours'),
                               description=_("The duration of the contribution"))
+
+
+class ContributionProtectionForm(IndicoForm):
+    managers = PrincipalListField(_('Managers'), description=_('List of users allowed to modify the contribution'))
+    protection_mode = IndicoRadioField(_('Protection mode'), orientation='horizontal',
+                                       choices=[('public', _('Public')), ('inheriting', _('Inheriting')),
+                                                ('protected', _('Protected'))],
+                                       description=_('Control the protection mode of the contribution'))
```

Comment:
This description seems pretty useless. It basically repeats what the title already says. I'd rather don't put any description at all or actually explain what this does